srai/joiners/intersection_joiner.py

```diff
@@ -0,0 +1,67 @@
+"""
+Intersection Joiner.
+
+This module contains intersection joiner implementation.
+
+"""
+
+from dataclasses import dataclass
+
+import geopandas as gpd
+import pandas as pd
+
+
+@dataclass
+class JoinerResult:
+    """Joiner result dataclass."""
+
+    regions: gpd.GeoDataFrame
+    features: gpd.GeoDataFrame
+    joined: gpd.GeoDataFrame
```

Comment:
This Joiner result could be extracted to a dedicated file since it will probably be shared between most joiners.

Reply:
You're right, moving it to `base.py`